dockers/docker-base-stretch/Dockerfile.j2

```diff
@@ -0,0 +1,82 @@
+FROM debian:stretch
+
+# Clean documentation in FROM image
+RUN find /usr/share/doc -depth \( -type f -o -type l \) ! -name copyright | xargs rm || true
+
+# Clean doc directories that are empty or only contain empty directories
+RUN while [ -n "$(find /usr/share/doc -depth -type d -empty -print -exec rmdir {} +)" ]; do :; done
+RUN rm -rf               \
+    /usr/share/man/*     \
+    /usr/share/groff/*   \
+    /usr/share/info/*    \
+    /usr/share/lintian/* \
+    /usr/share/linda/*   \
+    /var/cache/man/*     \
+    /usr/share/locale/*
+
+# Make apt-get non-interactive
+ENV DEBIAN_FRONTEND=noninteractive
+
+# Configure data sources for apt/dpkg
+COPY ["dpkg_01_drop", "/etc/dpkg/dpkg.cfg.d/01_drop"]
+COPY ["sources.list", "/etc/apt/sources.list"]
+COPY ["no_install_recommend_suggest", "/etc/apt/apt.conf.d"]
+RUN apt-get update
+
+# Pre-install fundamental packages
+RUN apt-get -y install \
+    less               \
+    perl               \
+    procps             \
```

Comment:
this is newly added compare with jessie, why?

Reply:
I added this because I found the `ps` command was missing from the default Stretch docker image.

Reply:
ok, i am fine.